mtest/libmscore/timesig/tst_timesig.cpp

```diff
@@ -224,7 +224,7 @@ void TestTimesig::timesig08()
 
       Measure* m1 = score->firstMeasure();
       Segment* seg = m1->findSegment(SegmentType::TimeSigAnnounce, m1->endTick());
-      Element* el = seg->element(4); // stave no.2
+      Element* el = seg->element(VOICES); // stave no.2
```

Comment:
Another option: `staff2track(1)` :)
https://github.com/musescore/MuseScore/blob/e44c823c070187fb29a49f21fb2793bda6b072f0/libmscore/mscore.h#L75

Reply:
But isn't this `<< 2` taking advantage of `VOICES` being 4? What if it's 5 or else? Speaking of this, I think directly putting `VOICES` is even more correct than this.

Reply:
Anyway, changed to `staff2track(1)`.

Reply:
> But isn't this `<< 2` taking advantage of `VOICES` being 4

Yes, but this is (or at least was intended to be) a commonly used function to convert between staff indices and track numbers, and its definition would definitely change if we ever decide to switch voices number or go to another track numbering scheme at all. That is why I mentioned this. Still this conversion is not really likely as too much depends on it so this is definitely not very important issue.